common/djangoapps/adaptive_learning/config/tests/utils.py

```diff
@@ -0,0 +1,25 @@
+"""
+Provides helper functions for tests which need
+to configure flags related to Adaptive Learning.
+"""
+from contextlib import contextmanager
+
+from adaptive_learning.config.models import (
+        AdaptiveLearningEnabledFlag,
+        CourseAdaptiveLearningFlag
```

Comment:
@tomaszgy Indentation should be 4 spaces here.

Reply:
👍